sidecars/shellcheck/Dockerfile

```diff
@@ -9,9 +9,11 @@
 #   Red Hat, Inc. - initial API and implementation
 FROM koalaman/shellcheck-alpine:v0.7.1
```

Comment:
We may want to update this to `v0.7.2` while we're here, the `v0.7.1` tag hasn't been updated in a year or so.